test/harry/main/org/apache/cassandra/harry/model/SelectHelper.java

```diff
@@ -140,7 +140,7 @@ public static CompiledStatement count(SchemaSpec schema, long pd)
     private static SchemaSpec.AddRelationCallback appendToBuilderCallback(Select.Builder builder)
     {
         return (spec, kind, value) ->
-               builder.withWhere(toInequalities(kind), Reference.of(new Symbol(spec.name, spec.type.asServerType())), new Bind(value, spec.type.asServerType()));
```

Comment:
style change... I think the code is easier to read if you push the kind in the middle... `col > 42` vs `> col 42`